ONBOARDING_PROJECT.md

```diff
@@ -28,16 +28,13 @@ The Wishlist item in the user wishlist should include:
 - Place
 - Title
 
-Bonus task, optional but recommended: Our app was originally designed to contact a single backend-for-frontend known as the *Native Apps Proxy*. The proxy contacted other services through various protocols, and merged and transformed results for an app-specific response. 
-As requirements and strategies have changed, we have moved to instead use generic services, mostly maintained by other teams at FINN, but offered through the *Apps Gateway* to give a single root URL and a uniform way of accessing backend services. 
+Expand 'FINNClient' with some 'WishlistItemsClient' protocol to load i.e. 'Response<FavoriteAdContainer>' and then parse it to necessary type. It can be taken i.e. from existed 'FavoriteItemsClient' to load user's favorite ads and work with it as with ads added to wishilst.
```

Comment:
This is specific to the iOS app, architecture and class structure is different for Android. Also, I think the bonus task could be kept in for those who feel up to the task.

Reply:
Oh, really, I thought these repo is only iOS specific. I'll update it now
Regarding removing off part about *Apps Gateway*, we've discussed it today on ios meeting and came to conclusion to replace it with Favorite-mock practice

Reply:
Changed